dependencies.R

```diff
@@ -1,4 +1,5 @@
 # install.packages(c(), repo="http://cran.rstudio.com/")
+install.packages("bookdown", repo="http://cran.rstudio.com/")
```

Comment:
shouldn't blogdown be already installed by the anaconda environment?

Reply:
Is bookdown included in blogdown? We added this line since we used bookdown for our citations.